ci-scripts/firebase_publish_report.py

```diff
@@ -57,9 +57,9 @@ def url_to_html_redirect(run_id: str, url: Optional[str]):
         report_url = f'https://github.com/magma/magma/actions/runs/{run_id}'
 
     return (
```

Comment:
nitpick: `return f'<script>window.location.href="{report_url}"</script>' should also work, looks slightly simpler to me and doesn't risk accidentally returning a tuple.

Reply:
Yes, it was split in three lines before so I did not want to change this

Reply:
another nitpick: We could annotate the return type of the function so mypy would catch this.